pom.xml

```diff
@@ -226,6 +226,22 @@
                     <repoToken>${COVERALL_TOKEN}</repoToken>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
```

Comment:
We don't need it on the main branch as much because you have the release available through maven. Absent of a single new release, we need a way to build fat jars. It probably should not be in your repo. I will take it out and resubmit as a PR to your branch.